products/health-care/appointments/va-online-scheduling/research/july-2020-user-research/vaosr-june-research-plan.md

```diff
@@ -1,50 +1,86 @@
+1. Participant criteria: What are you looking for in a participant?	
+   - 20	
+   - Ideal, but not required: individuals who need to or who have recently scheduled an appointment
+
+2. What is your recruitment strategy? 	
+   Perigean	
+
 ## When? 	
 1.	Timeline: What dates do you plan to do research? 	
 (IF you are using the research recruiting contract, please submit 1 FULL week prior to the start of research for remote, 2+ weeks for in person.) 	
 2.	Prepare: When will the thing you are testing be ready? (Goes without saying, but should be a few days before testing will begin.) 	
 3. Length of Sessions: How long do you estimate each session will be? (This helps with scheduling & thank you gifts.) e.g. 30 minutes, < 1 hour, up to 2 hours, up to 4 hours) 	
 4.	Availability: If applicable, when would you like sessions scheduled? **Please list exact dates and times in EASTERN Standard Time**. Please request enough dates and time slots (e.g. Monday 9-1, 3-6; Tuesday 9-6, etc.). Be as flexible as possible, cognizant that many Veterans are only available before and after working times, and live across the U.S.	
 5.	Pilot: Please indicate a date before your sessions begin for piloting your research. Which member of the design team will you pilot your research with? 	
+
 ## Team Roles	
 Please list the people who will be serving in each role. **Include the primary phone number for moderator and the emails for moderator, notetaker, and observers. If you need Perigean to take notes for you, indicate that next to Notetaker** 	
-- Moderator:	
-- Research guide writing and task development (usually but not always same as moderator):	
-- Participant recruiting & screening:	
-- Project point of contact:	
-- Participant(s) for pilot test:	
-- Note-takers:	
-- Observers:	
-**List email addresses for those who should attend and observe the sessions: VA Stakeholders, engineering team members, design team members, any other people who might find this research relevant to their work**	
+- Moderator: Melissa Schaff ([melissa.j.schaff2@omb.eop.gov](mailto:melissa.j.schaff@omb.eop.gov))
+- Research guide writing and task development (usually but not always same as moderator): Melissa Schaff ([melissa.j.schaff2@omb.eop.gov](mailto:melissa.j.schaff@omb.eop.gov))
+- Participant recruiting & screening: Perigean 
+- Project point of contact: Melissa Schaff ([melissa.j.schaff2@omb.eop.gov](mailto:melissa.j.schaff@omb.eop.gov))
+- Participant(s) for pilot test: Melissa Schaff ([melissa.j.schaff2@omb.eop.gov](mailto:melissa.j.schaff@omb.eop.gov))
+- Note-takers: Perigean
+- Observers: 
+  Melissa Schaff ([melissa.j.schaff2@omb.eop.gov](mailto:melissa.j.schaff@omb.eop.gov))
+  Lauren Alexanderson ([lauren.alexanderson@va.gov](mailto:lauren.alexanderson@va.gov))
+  Ryan Thurwell ([ryan.thurlwell@va.gov](mailto:ryan.thurlwell@va.gov))
+  Lenae Storey ([lenae.storey@adhocteam.us](mailto:lenae.storey@adhocteam.us))
+
```

Comment:
I want to invited the eng team to these as well. do i need to list all of those folks?

Reply:
They may not come to all or many, but i want them to observe when they can.

Reply:
yes, please add them, so they can be included